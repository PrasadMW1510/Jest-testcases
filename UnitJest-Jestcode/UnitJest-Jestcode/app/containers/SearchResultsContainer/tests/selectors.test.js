import { fromJS } from 'immutable';
import makeSelectSearchResultsContainer, { getselectedBookingResults } from '../selectors';

describe('search Results Container selector', () => {
  it('should select the search Results', () => {
    const searchResultsContainer = fromJS({
      searchResultsContainer: { name: 'test' },
    });
    const mockedState = fromJS({
      searchResultsContainer,
    });

    expect(makeSelectSearchResultsContainer()(mockedState)).toEqual(searchResultsContainer.toJS());
    expect(getselectedBookingResults()(mockedState)).toMatchSnapshot();
  });
});
