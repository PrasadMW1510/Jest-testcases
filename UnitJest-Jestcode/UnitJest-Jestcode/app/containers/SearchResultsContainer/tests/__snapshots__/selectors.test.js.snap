// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search Results Container selector should select the search Results 1`] = `
Immutable.Map {
  searchResultsContainer: Immutable.Map {
    name: "test",
  },
}
`;
